pkg/types/core/capabilities_registry.go

```diff
@@ -17,4 +17,5 @@ type CapabilitiesRegistry interface {
 	GetTarget(ctx context.Context, ID string) (capabilities.TargetCapability, error)
 	List(ctx context.Context) ([]capabilities.BaseCapability, error)
 	Add(ctx context.Context, c capabilities.BaseCapability) error
+	Remove(c capabilities.BaseCapability) error
```

Comment:
I wonder how we make sure that malicious/buggy capabilities don't add/remove entries for others. Maybe we don't have to? Can we say it's node's responsibility to run legitimate capabilities...?

Reply:
> it's node's responsibility to run legitimate capabilities
I agree with this with regards to 3rd party capabilities. I imagine there would be:
- a code signature of the verified source code. Maybe this is something we add to the Capability Registry?
- more testing rails on what is allowed, e.g. ensure there are no conflicting Capability `Info`s and run LOOPP initialize function on a mock CapRegistry
- some official stamp of "CLL reviewed"

Reply:
We should be able to solve this for standard capabilities by moving the removal to the core node; for the OCR3 capability that's a bit more difficult though and we'll need to do it via the loopp boundary